metrics/web/lib/auth/presentation/state/auth_notifier.dart

```diff
@@ -77,8 +78,8 @@ class AuthNotifier extends ChangeNotifier {
   /// during loading user profile data.
   PersistentStoreErrorMessage _userProfileErrorMessage;
 
-  /// Contains a user's authentication status.
-  bool _isLoggedIn;
+  /// Contains a state of the user authorization.
```

Comment:
🏷️ 
```suggestion
  /// Contains a user's authorization state.
```
What do you think?